tldw_Server_API/app/core/Ingestion_Media_Processing/claims_utils.py

```
@@ -0,0 +1,269 @@
+        if details is not None:
+            details.setdefault("stored_in_db", 0)
+            process_result["claims_details"] = details
+        return
+
+    def _worker() -> int:
+        db = MediaDatabase(db_path=db_path, client_id=client_id)
+        try:
+            try:
+                db.soft_delete_claims_for_media(int(media_id))
+            except Exception:
+                pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)